client/src/components/ImageCarousel.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import $ from 'jquery';
+import ImageListEntry from './ImageListEntry.jsx';
+import ImageRotateButton from './ImageRotateButton.jsx';
+
+class ImageCarousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imageList: [],
+      centerImageIdx: 0,
+    };
+  }
+
+  componentDidMount() {
+    $.ajax({
+      url: 'http://localhost:3001/businesses/1/images',
+      method: 'GET',
+      success: (imageList) => {
```

Comment:
I'm not sure if it would be better, but you might consider breaking some of this into helper functions.  I did that to make it easier to write different kinds of tests, but ultimately it does the same thing. For example, a fetch method, and then an update state method.  Just a thought.